tests/ruby/RuboCopBearTest.py

```diff
@@ -1,57 +1,91 @@
 import os
+from queue import Queue
+
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Section import Section
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
 
 from bears.ruby.RuboCopBear import RuboCopBear
-from coalib.testing.LocalBearTestHelper import verify_local_bear
 
-good_file = """def good_name
```

Comment:
why do we need to move the text out into real filenames?

Does rubocop require that there is a valid file with this filename?

Cant `LocalBearTestHelper` create temporary files with an appropriate filename/extension?

If it cant, then writing tests is much harder, which is sort of the anti-thesis of your project.